devops/README.md

```diff
@@ -0,0 +1,34 @@
+# Deployment
```

Comment:
on the top-level README.md, we have
> Kubernetes (Production/Staging)
Information on Kubernetes configuration for production and staging environments can be found in the [kube/README.md](https://github.com/TACC-Cloud/geoapi/blob/master/kube/README.md) including information on kube commands and Jenkins deployment workflows.

can we update that README.md and point that to `devops/README.md`?